src/include/sof/samples/audio/c_models_inc/config.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
```

Comment:
@cristinailie we should find a fancy name for this KWD feature and prefix every macro/variable/function in the global context with it.

Also, I would suggest renaming c_models_inc to c_models. It is clear that this directory will contain header files because it is in the include directory.

Unless, somehow you took this directory from another framework and we want to keep the compatibility.

You have better information about the inner details of this feature. Maybe the prefix should be

KWD_NN (keyword detection using neural networks).